tests/integration/cli/test_utils.py

```diff
@@ -40,7 +39,7 @@ def test_can_split_video(self):
             "--start-frame",
             "2",
             "--end-frame",
-            "8",
+            "7",
```

Comment:
I guess there are some non-deterministic processes during video encoding and decoding. Can we make video frame grabbing deterministic using `mock.patch`? For example,
```suggestion
with mock.patch("datumaro.components.media.VideoFrame.data") as mock_video_frame_data:
     mock_video_frame_data.side_effect = [np.full(value=idx, size=(3,32,32), dtype=np.uint8) for frame_idx in range(10)]
     run(
         ...
     )
```

Reply:
Thanks Vinnam, I have updated the test as following your guide. Please review again when you are available.